README.MD

```diff
@@ -4,7 +4,7 @@
 
 The official C# SDK for the [Model Context Protocol](https://modelcontextprotocol.io/), enabling .NET applications, services, and libraries to implement and interact with MCP clients and servers.
 
-> [!NOTE]  
+> [!NOTE]
```

Comment:
Sorry about the whitespace changes here -- VSCode automatically strips trailing whitespace.

I can revert these if needed.

Reply:
The whitespace deletions are fine.